collector/filesystem_openbsd_amd64.go

```diff
@@ -11,7 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build openbsd darwin,amd64 dragonfly
+// +build openbsd
```

Comment:
@jamesrwhite it seems like having darwin/dragonfly on this build was a mistake, so I've removed it.

Reply:
Makes sense, I was a bit confused why they were there 😅